mitodl/odl-video-service
# Show collection and video titles

#### What are the relevant tickets?
Fixes #543 
Fixes #457 

#### What's this PR do?
Adds the collection title to collection page header
Adds the video title to video detail & embed page headers

#### How should this be manually tested?
- Go to the home page (`/collections`)
- Click on a collection.  The browser tab title should change to `OVS | <collection title>`
- Click on a video.  The browser tab title should change to `OVS | <video title> | Video Detail`
- Add `/embed` to the video URL and load it.  The browser tab title should change to `OVS | <video title> | Video Embed`
